listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -516,6 +516,14 @@ export default class RecentListens extends React.Component<
     return recordingMsid ? _.get(recordingFeedbackMap, recordingMsid, 0) : 0;
   };
 
+  updateListenList = (listen: Listen) => {
+    const { listens } = this.state;
```

Comment:
This function name could be more descriptive. Update how?

`removeListenFromListenList`?

Reply:
done